library/entity/entity_networking/src/testmod/java/org/quiltmc/qsl/entity/networking/test/mixin/CreeperEntityMixin.java

```diff
@@ -66,4 +74,35 @@ protected CreeperEntityMixin(EntityType<? extends HostileEntity> entityType, Wor
 			}
 		}
 	}
+
+	// Make creepers drop a random item on explosion and render it over their head to test extended spawn data
+
+	private ItemStack quiltTestMod$stackToDrop;
```

Comment:
oh dear, this would be needed
```suggestion
	@Unique
	private ItemStack quilt$stackToDrop;
```